bigquery_etl/cli/udf.py

```diff
@@ -57,15 +58,19 @@ def _udfs_matching_name_pattern(pattern, udf_paths):
 def udf(ctx):
     """Create the CLI group for the UDF command."""
     ctx.ensure_object(dict)
-    ctx.obj["UDF_DIRS"] = ("udf", "udf_js")
+    # todo: make generic for projects
+    ctx.obj["UDF_DIRS"] = (
+        "sql/moz-fx-data-shared-prod/udf",
+        "sql/moz-fx-data-shared-prod/udf_js",
+    )
```

Comment:
This will be a follow-up PR which will also address https://github.com/mozilla/bigquery-etl/issues/1342